simulation/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "simulation"
+version = "0.1.0"
+edition = "2021"
+authors = ["DatenLord <dev@datenlord.io>"]
+repository = "https://github.com/datenlord/Xline/tree/master/tests/simulation"
+description = "Madsim simulation for Xline"
+license = "Apache-2.0"
+readme = "README.md"
+categories = ["Test"]
+keywords = ["Test", "Simulation"]
```

Comment:
IMO, deterministic simulation is better.

Reply:
done